src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -303,6 +357,57 @@ public static boolean isOfType(DetailAST ast, int... types) {
         return ast != null && isOfType(ast.getType(), types);
     }
 
+    /**
+     * Determines if the AST belongs to the given first parameter.
+     *
+     * @param ast the AST node to check
+     * @param first the acceptable token
+     *
+     * @return true if type matches one of the given parameter.
+     */
+    public static boolean isOfType(DetailAST ast, int first) {
+        return ast != null && isOfType(ast.getType(), first);
+    }
+
+    /**
+     * Determines if the AST belongs to the given first and second parameters.
+     *
+     * @param ast the AST node to check
+     * @param first the acceptable token
+     * @param second the acceptable token
+     * @return true if type matches one of the given parameter.
```

Comment:
`true if type matches one of the given parameter.`
-->
`true if type matches one of the given types.`

Reply:
Changes Done